vignettes/Demography.Rmd

```diff
@@ -15,26 +15,21 @@ knitr::opts_chunk$set(
 
-The dynamics of malaria transmission, and the efficacy of interventions designed to interrupt it, are highly context specific and an important consideration is the demography of the population(s) under investigation. To suit the research needs of the user, `malariasimulation` allows for the specification of custom human population demographics by allowing them to specify age-group specific death rates parameters. It also enables users to instruct the model to render outputs with variables of interest (e.g. human population, number of cases detectable by microscopy, number of severe cases, etc.) presented by user-defined age groups. In this vignette, we use simple, illustrative cases to demonstrate how to customise the output rendered by the `run_simulation()` function, and how to specify both fixed and time-varying custom death rates.
-
-# Specifying human death rates by age group
-
-The `malariasimulation` package allows users to capture different human population demographies by specifying age-specific death rates. In the first section, we'll demonstrate how to instruct the model to output population and disease metrics by user-defined age groups and how to establish and run simulations with age-specific death rates. We'll illustrate the effect this has by comparing the demographic make-up of this custom parameterisation with that produced using the demographic profile established by the default parameters.
-
-## Default Parameterisation
+## Age group rendering
 
 First, we'll establish a base set of parameters using the `get_parameters()` function and accept the default values. The `run_simulation()` function's default behaviour is to output only the number of individuals aged 2-10 years old (`output$n_730_3650`, where 730 and 3650 are the ages in days). However, the user can instruct the model to output the number of individuals in age groups of their choosing using the `age_group_rendering_min_ages` and `age_group_rendering_max_ages` parameters. These arguments take vectors containing the minimum and maximum ages (in daily time steps) of each age group to be rendered To allow us to see the effect of changing demographic parameters, we'll use this functionality to output the number of individuals in ages groups ranging from 0 to 85 at 5 year intervals. Note that the same is possible for other model outputs using their equivalent min/max age-class rendering arguments (`n_detect` , `p_detect`, `n_severe`, `n_inc`, `p_inc`, `n_inc_clinical`, `p_inc_clinical`, `n_inc_severe`, and `p_inc_severe`, run `?run_simulation()` for more detail).
```

Comment:
Add a full stop between "rendered" and "To allow" (fourth line as it appears on screen here)